applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -58,6 +60,28 @@ return view.extend({
 		o = s.option(form.Flag, "debug", _("Enable debug logging"));
 		o.rmempty = false;
 
+		if (ddnsDomains) {
+			let ddnsDomainsList = [];
+			for (let ddnsDomain of ddnsDomains) {
+				ddnsDomainsList.push(ddnsDomain.domains[0]);
+			}
+			o = s.option(form.Button, '_import_ddns');
+			o.title = _('Found DDNS domains');
+			o.inputtitle = _('Import') + ': ' + ddnsDomainsList.join();
+			o.inputstyle = 'apply';
+			o.onclick = function () {
+				for (let ddnsDomain of ddnsDomains) {
+					let sectionId = uci.add('acme', 'cert', ddnsDomain.name);
+					uci.set('acme', sectionId, 'domains', ddnsDomain.domains);
+					uci.set('acme', sectionId, 'validation_method', 'dns');
+					uci.set('acme', sectionId, 'dns', ddnsDomain.dnsApi);
+					uci.set('acme', sectionId, 'credentials', ddnsDomain.credentials);
+				}
+				uci.save();
+				window.location.reload();
+			};
```

Comment:
While this is neat, I also worry that it's a bit on the "magic" side. At the very least, we should protect against a user accidentally clicking the import button multiple times and ending up with multiple identical configuration sections. Probably adding some help text to explain what the button does, so a user can see this before clicking it would be nice :)

Reply:
Right, but then we should explain that in the help text (with a note like "look at the sections added below after clicking this to make sure things look right" or something to that effect). And we can still check for duplicates before just adding a bunch of new sections...